src/commands/PublishCommand.js

```diff
@@ -349,6 +349,9 @@ export default class PublishCommand extends Command {
   }
 
   npmPublishAsPrerelease(callback) {
+    const {skipTempTag} = this.getOptions();
+    // if we skip temp tags we should tag with the proper value immediately therefore no updates will be needed
+    const tag = !skipTempTag ? "lerna-temp" : this.getDistTag();
```

Comment:
Let's invert the ternary (`const tag = skipTempTag ? this.getDistTag() : "lerna-temp";`) and make sure there's a newline between this assignment and the next `forEach` block.